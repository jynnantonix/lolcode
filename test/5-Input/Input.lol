This test checks to see whether a program correctly accepts input via the GIMMEH statement.  It keeps asking for input until the user enters QUIT.

HAI
	I HAS A var
	GIMMEH var
	IM IN YR loop WILE DIFFRINT var AN "QUIT"
		VISIBLE var MKAY?
		GIMMEH var
	IM OUTTA YR loop
KTHXBAI
