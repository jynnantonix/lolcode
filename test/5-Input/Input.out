LOL>> 4 
LOL>> 34 
LOL>> Hello 
LOL>> this 
LOL>> is output 
LOL>> with 
LOL>> the LOL>> input command 
LOL>> 